website/docs/basics/blocks-attributes.md

```diff
@@ -12,3 +12,90 @@ The reason for this is that you can't control what key of that object is stored
 You can also look at this from the Reacts perspective: setting the entire object every time the key changes is bad for performance.
 
 Yes, you can use objects, but we recommend using them only when you want to store multiple keys simultaneously.
+
+### Structure 
+
+All attributes in the block/component must begin with the exact same prefix as it is defined in the `blockName` or `componentName`.
```

Comment:
Maybe we should explain why after the examples.

Something along the lines of: We use this naming so that our helpers can internally resolve attributes (or whatever that black magic is 😂)

Reply:
agreed

Reply:
added